.github/workflows/ci.yml

```diff
@@ -0,0 +1,86 @@
+name: ci
+on: [push, pull_request]
+env:
+  COVERAGE_OPTION: ./node_modules/.bin/nyc
+  NODE_VERSION: 10
+  PARSE_SERVER_TEST_TIMEOUT: 20000
+jobs:
+  check-mongo:
+    strategy:
+      matrix:
+        include:
+          - name: Mongo 4.0.4, ReplicaSet, WiredTiger
+            MONGODB_VERSION: 4.0.4
+            MONGODB_TOPOLOGY: replicaset
```

Comment:
Mongo 4.0.0 was released in Aug 6, 2018, we probably need to add also 4.2/4.4 (i use these versions in prod and do not notice any issue)

Reply:
done